app/views/searches/index.slim

```diff
@@ -0,0 +1,37 @@
+.row-flex.pt-3
+  .col-2
+  .col-8
+    h2 Search
+
+    = form_tag search_path, method: :get do
+      .input-group.mb-3
+        = text_field_tag :search_query, params[:search_query], class: 'form-control', placeholder: 'Find a tutor...'
+        .input-group-append
+          = submit_tag "Search", name: nil, class: 'btn btn-primary'
+
+    - @profiles.each do |profile|
```

Comment:
Use hepler `profiles` like described in comment above, and move `profile` to the partial, so that it would be
`render partial: 'profile', collection: profiles`